scripts-components/in-view/tsconfig.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
Zvážil bych použití [`extends`](https://www.typescriptlang.org/docs/handbook/tsconfig-json.html#configuration-inheritance-with-extends) místo kopírování+duplikování obsahu tsconfigu.

Reply:
Skvělý nápad. Díky. Done.